src/qml/test/tst_onboarding.qml

```diff
@@ -0,0 +1,60 @@
+import QtQuick 2.15
+import QtTest 1.15
+import "../controls"
+import "../pages/onboarding"
+
+Rectangle {
+    id: root
+    width: 640
+    height: 665
+    color: Theme.color.background
+
+    OnboardingWizard {
+        id: onboardingWizard
+        anchors.fill: parent
+    }
+
+    TestCase {
+        id: onboardingFlow
+        when: windowShown
+
+        function test_all_continue_buttons() {
+            verify(onboardingWizard.currentIndex == 0)
+ 
```

Comment:
Needed to make `./test/lint/lint-whitespace.py` happy:
```suggestion

```

In general, suggesting ensure that linters are happy for each commit.